packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -942,11 +964,12 @@ export function TransferPanel() {
       setTransferring(true)
```

Comment:
It would be nice to check for `transferMode === 'unsupported'` just before L954 and throw just in case

Reply:
i also created a followup ticket to handle error messaging in `useTransferReadiness` even though this case should never be hit on the UI

https://linear.app/offchain-labs/issue/FS-1100/include-transfermode-===-unsupported-in-usetransferreadiness